src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age ? <p className="Person__age">I am {person.age}</p> : ''}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
+    )}
```

Comment:
Remember about DRY rule(https://ru.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself)

Now you use `<p className="Person__partner">` twice, refactor this part of code